meta-mentor-staging/recipes-multimedia/mplayer/mpv_0.15.0.bb

```diff
@@ -4,6 +4,7 @@ HOMEPAGE = "http://www.mpv.io/"
 DEPENDS = "zlib ffmpeg jpeg virtual/libx11 xsp libxv \
            libxscrnsaver libv4l libxinerama \
            libxkbcommon \
+           pulseaudio mesa \
```

Comment:
yes, or maybe virtual/libgl would be more appropriate, since MPV actually needs libgl that mesa provides.